app/src/main/res/layout/activity_main.xml

```diff
@@ -33,4 +33,14 @@
         app:layout_constraintTop_toTopOf="parent"
         app:navGraph="@navigation/mobile_navigation" />
 
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/app_name"
```

Comment:
Instead of adding another widget to the screen, I'd use the Action Bar to show the app name.  This is more common, and it's where users expect to see the application's name.